databases/core.py

```diff
@@ -278,7 +278,7 @@ async def execute_many(
     ) -> None:
         queries = [self._build_query(query, values_set) for values_set in values]
         async with self._query_lock:
-            await self._connection.execute_many(queries)
+            await self._connection.execute_many(queries, values)
```

Comment:
Should we avoid building the queries here if you're going to compile them later?

Reply:
Hmm, it's been a long time since I wrote this PR, but I think we still need to convert the query to ClauseElement and bind params, but maybe we don't need list comprehension here, so maybe something like that is sufficient
```
async def execute_many(
        self, query: typing.Union[ClauseElement, str], values: list
    ) -> None:
        query = self._build_query(query,values[0])
        async with self._query_lock:
            await self._connection.execute_many(query, values)
```

Reply:
Hm interesting. I'm not sure myself, just wanted to check if you had considered it.

Thanks for chiming in on this after the delay. If you're interested in working on it again I can review and release it. Did you confirm this works and improves performance as expected?

Reply:
I can work on it tmr for a few.
it should improve the performance but i didn't make any benchmarks tbh